app/ui-sidenav/client/sideNav.js

```diff
@@ -39,6 +39,9 @@ Template.sideNav.helpers({
 	},
 
 	sidebarViewMode() {
+		if (/Mobi/.test(navigator.userAgent)) {
```

Comment:
Can we create a helper method to determine if the client is a mobile client and use that helper function everywhere?